docusaurus/docs/iOS/guides/working-with-the-composer.md

```diff
@@ -0,0 +1,221 @@
+        case .new:
+            composerView.inputMessageView.textView.placeholderLabel.text = "iMessage"
+        default:
+            break
+        }
+    }
+}
+
+// Then, replace the components:
+components.messageComposerVC = iMessageComposerVC.self
+components.messageComposerView = iMessageComposerView.self
+```
+
+**Result:**
+<img src={require("../assets/composer-ui-custom-layout.png").default} width="50%"/>
+
+Since the Composer is built with multiple `ContainerStackView`'s with a similar API of a `UIStackView`, it is very easy to change the Composer layout it is just a question of removing views from containers and adding them back to the desired containers. In this case, we override the `setUpLayout()` lifecycle to remove the `commandsButton` all together and we move the `sendButton` from the `trailingContainer` to the `inputTextContainer`. As a last touch, we change the `attachmentButton` image by overriding the `setUpAppearance()` lifecycle.
+
+## Customizing the Composer Actions
+All the actions triggered by the Composer buttons are wired in the `ComposerVC.setUp()` lifecycle. When adding new buttons it is possible to add new actions by overriding this function. Let's imagine an example where we have added a new custom emoji button to open an emoji picker:
+```swift
+final class CustomComposerVC: ComposerVC {
+    override func setUp() {
+        super.setUp()
+
+        composerView.emojiButton.addTarget(self, action: #selector(showEmojiPicker), for: .touchUpInside)
+    }
+
+    @objc open func showEmojiPicker(sender: UIButton) {
+        // Custom implementation of opening the emoji picker
+    }
+}
+
+components.messageComposerVC = CustomComposerVC.self
+```
+
+Besides adding new actions, it is also possible to customize the existing actions, for example, an application might want to present the attachments picker on a completely different view and with different logic:
+```swift
+final class CustomComposerVC: ComposerVC {
+    override func showAttachmentsPicker(sender: UIButton) {
+        // Custom implementation of opening the attachments picker
+    }
+}
+
+components.messageComposerVC = CustomComposerVC.self
+```
+
+## Customizing Typing Suggestions
+When typing on the composer, if you type `@` it will present a view suggesting users that you can mention, or if you type `/` as the first character, it will show command suggestions. You can customize not only the view of the suggestions but also the logic around it.
+
+### Changing the Recognized Symbol
+If your app has a different character to recognize user mentions or commands, then you just need to override the symbols like this:
+```swift
+final class CustomComposerVC: ComposerVC {
+    override var mentionSymbol: String { "👤" }
+
+    override var commandSymbol: String { "⚡️" }
+}
+
+components.messageComposerVC = CustomComposerVC.self
+```
+That's it, with this in place, the suggestions will now be triggered when you type your custom symbol.
```

Comment:
Let's remove this section completely now, or move it to the bottom, it's not very relevant

Reply:
Yes makes sense, let's remove it, since anyone can understand this just by looking at the code 👍